tests/monotouch-test/Metal/MTLDeviceTests.cs

```diff
@@ -272,9 +272,7 @@ public void ReturnReleaseTest ()
 			}
 
 			if (TestRuntime.CheckXcodeVersion (10, 0)) {
-#if __MACOS__
-				if (device.SupportsFeatureSet (MTLFeatureSet.macOS_GPUFamily2_v1))
-#endif
+				if (device.SupportsFamily (MTLGpuFamily.Common2))
```

Comment:
minor: indentation
maybe move it with the `TestRuntime` check earlier (so no other lines needs to be changed)

Reply:
Fixed.